x-pack/packages/kbn-elastic-assistant-common/scripts/openapi/bundle.js

```diff
@@ -7,12 +7,13 @@
 
 require('../../../../../src/setup_node_env');
 const { bundle } = require('@kbn/openapi-bundler');
+// eslint-disable-next-line import/no-nodejs-modules
```

Comment:
Is there a higher-level way to get eslint to treat these files the same as they were previously? It seems like there's a discrepancy between how it treats plugin code vs package code. Unless this code _could_ be run in a browser, and `eslint` is correct?

Reply:
That's a good question! 🙂 I noticed the same thing and wasn't sure so was just following what folks had done in the other scripts. I thought it was interesting that the `osquery` generate script (which is in a plugin) used `@typescript-eslint/no-var-requires`:

<img width="1173" alt="image" src="https://github.com/elastic/kibana/assets/2946766/68a1ef02-dbc2-4296-8753-fb067606fe1b">


Whereas the `security_solution` generate script (also a plugin) didn't need it. And then of course as you mentioned, the package variants require `import/no-nodejs-modules`.

---

Perhaps @jbudz can provide some insight here when he stops by for the operations review?